api/audit/tasks.py

```diff
@@ -184,3 +185,46 @@ def create_segment_priorities_changed_audit_log(
             else timezone.now()
         ),
     )
+
+
+@register_task_handler()
+def create_version_rolled_back_audit_log_records(
+    rolled_back_to_uuid: str,
+    rolled_back_from_uuid: str,
+    user_id: int | None,
+    api_key_id: int | None,
+) -> None:
+    rolled_back_from = EnvironmentFeatureVersion.objects.select_related(
+        "feature", "feature"
+    ).get(uuid=rolled_back_from_uuid)
```

Comment:
Why is feature repeated twice here?

Reply:
Good catch, this was because I originally intended to add `"feature__project"` as well, but decided against it... apparently I made that decision halfway through writing the argument 😄